plugins/kubernetes/app/models/kuber_deploy_service.rb

```diff
@@ -46,6 +47,20 @@ def create_services!
     end
   end
 
+  def delete_replication_controllers!
+    kuber_release.release_docs.each do |release_doc|
+      log 'deleting existing ReplicationControllers', role: release_doc.kubernetes_role.name
+      labels = {project: release_group.build.project_name, role: release_doc.kubernetes_role.label_name}
+      client.get_replication_controllers(
+          label_selector: labels.to_kuber_selector,
+          namespace: kuber_release.deploy_group.cluster_deploy_group.namespace).each do |rc|
+        rc.spec.replicas = 0
+        client.update_replication_controller rc
+        client.delete_replication_controller rc.metadata['name'], rc.metadata['namespace']
+      end
+    end
+  end
```

Comment:
Yup, exactly. I didn't want to focus much on other strategies, the main goal is to not have 50 pods trying to run in my local cluster after I do some tests :)